samples/chapter-verse/proposed.json

```diff
@@ -0,0 +1,115 @@
+{
+  "type": "JOUST",
+  "version": "0.0.1-alpha.1",
+  "content": [
+    {
+      "type": "book:id",
+      "code": "MAT",
+      "text": "41MATGNT92.SFM, Good News Translation, June 2003"
+    },
+    {
+      "type": "chapter:c",
+      "number": "1",
+      "sid": "MAT 1"
+    },
+    {
+      "type": "para:cl",
+      "content": [
+        "Matthew",
+        {
+          "type": "char:ca",
+          "status": "invalid",
+          "content": [
+            "2"
+          ]
+        }
+      ]
+    },
+    {
+      "type": "para:cp",
+      "content": [
+        "M"
+      ]
+    },
+    {
+      "type": "para:cd",
+      "content": [
+        "Additional deacription about the chapter"
+      ]
+    },
+    {
+      "type": "para:s1",
+      "content": [
+        "The Ancestors of Jesus Christ"
+      ]
+    },
+    {
+      "type": "para:r",
+      "content": [
+        "(Luke 3.23-38)"
+      ]
+    },
+    {
+      "type": "para:p",
+      "content": [
+        {
+          "type": "verse:v",
+          "number": "1",
+          "sid": "MAT 1:1"
+        },
+        {
+          "type": "char:va",
+          "content": [
+            "3"
+          ]
+        },
+        {
+          "type": "char:vp",
+          "content": [
+            "1b"
+          ]
+        },
+        " This is the list of the ancestors of Jesus Christ, a descendant of David, who was a descendant of Abraham."
+      ]
+    },
+    {
+      "type": "chapter:c",
+      "number": "2",
+      "sid": "MAT 2"
+    },
+    {
+      "type": "para:p",
+      "content": [
+        {
+          "type": "verse:v",
+          "number": "1",
+          "sid": "MAT 2:1"
+        },
+        "\u0d15\u0d4d\u0d30\u0d3f\u0d38\u0d4d\u0d24\u0d41\u0d2f\u0d47\u0d36\u0d41\u0d35\u0d3f\u0d28\u0d4d\u0d31\u0d46 \u0d2c\u0d26\u0d4d\u0d27\u0d28\u0d3e\u0d2f ... ",
```

Comment:
Tiny stylistic thing but, another time, maybe use samples with characters that render in JSON? It's bad that, in 2023, some languages are more equal than others, but strings of unicode code points are not easy to eyeball.